src/igvfd/schemas/tissue.json

```diff
@@ -77,6 +88,14 @@
                 "week"
             ]
         },
+        "ccf_id": {
+            "title": "Common coordinate framework identifier",
+            "description": "This identifier associates an entry from the HubMap Consortium for specimens, biological structures, and spatial positions.",
```

Comment:
can we please change description to : "HubMap Common Coordinate Framework unique identifier corresponding to the organ, biological structure, and spatial location of the tissue specimen."

Reply:
Done